src/dev-core/src/app/tag/tag.demo.html

```diff
@@ -0,0 +1,36 @@
+<h1>Color Options</h1>
+<cwc-tag readonly color="gray">Austin</cwc-tag>
+<cwc-tag readonly color="purple">New York</cwc-tag>
+<cwc-tag readonly color="blue">Palo Alto</cwc-tag>
+<cwc-tag readonly color="orange">San Francisco</cwc-tag>
+<cwc-tag readonly color="light-blue">Seattle</cwc-tag>
+<cwc-tag readonly alias="1">Austin</cwc-tag>
+<cwc-tag readonly alias="2">New York</cwc-tag>
+<cwc-tag readonly alias="3">Palo Alto</cwc-tag>
+<cwc-tag readonly alias="4">San Francisco</cwc-tag>
+<cwc-tag readonly alias="5">Seattle</cwc-tag>
```

Comment:
This is more of an API design question, is there a reason to have two different APIs for color? Seems like we could simplify the Web Component API from the existing angular/ui API by providing just one and drop `alias`. Same with badges. I may be missing some historical context why alias existed in the previous APIs.

Reply:
It's a good question. I don't know the historical context for the alias or whether applications use this. If no good reason, I'm in agreement that we could remove this for core. Angular/UI might need to go through proper deprecation cycle for removal.. Maybe @mathisscott can shed some light on this?

Reply:
I would vote to drop the `alias` api for core but not going to block the PR on it. We can make sure to have a decision before 3.0